README.md

```diff
@@ -38,13 +41,156 @@ class Translations {
 }
 ```
 
-### Flavors
+Did you notice the `@GenerateArb()` annotation? That where the magic come from.
```

Comment:
Did you notice the `@GenerateArb()` annotation? That where the magic come from. 🪄